example/template.yaml

```diff
@@ -21,8 +21,6 @@ Resources:
       Environment: # More info about Env Vars: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#environment-object
         Variables:
           POWERTOOLS_SERVICE_NAME: "Payment Service"
-          LOG_LEVEL: INFO
```

Comment:
Any specific reason for removing this ?

Reply:
no, it was removed due to merge with master to resolve conflicts, I will add it back.